appengine/flexible/AntiForgery/README.md

```diff
@@ -0,0 +1,54 @@
+# AntiForgery Sample for Google App Engine Flexible Environment
+
+This sample application is ready to deploy to Google App Engine Flexible Environment.
+
+## Prerequisites
+
+1.  **Follow the set-up instructions in [the documentation](https://cloud.google.com/dotnet/docs/setup).**
+
+2.  Install the [Google Cloud SDK](https://cloud.google.com/sdk/).  The Google Cloud SDK
+    is required to deploy .NET applications to App Engine.
+
+3.  Install the [.NET Core SDK, version 2.0](https://github.com/dotnet/core/blob/master/release-notes/download-archives/2.0.5-download.md)
+    or newer.
+
+
+## ![PowerShell](../.resources/powershell.png) Using PowerShell
```

Comment:
None of this part looks like it's PowerShell-specific, and it could be off-putting... possibly include "or the standard Windows Command Prompt" or similar?

That said, another comment I was going to make is that there's nothing about running the Set-Up.ps1 script - is that run automatically as part of gcloud beta app deploy, but only when running on PowerShell?

(It would be interesting to know how many of our users are happy running samples with PowerShell... possibly something we should poll in a survey at some point.)

Reply:
Added discussion of Set-Up.ps1 and Clean-Up.ps1.